database/apollo.sql

```diff
@@ -0,0 +1,506 @@
+CREATE EXTENSION citext;
```

Comment:
Once we've got this schema into something that can manage it (i.e. Flyway) lets break it down and separate out schema primitives (like types and tables) from functions (whose migrations can be run repeatedly without the need to create new versions every time you update the function).